Kernel/Assertions.cpp

```diff
@@ -1,25 +1,19 @@
 /*
  * Copyright (c) 2018-2020, Andreas Kling <kling@serenityos.org>
+ * Copyright (c) 2023, Konrad <konrad@serenityos.org>
```

Comment:
Do you actually have this email address?

Reply:
nope. i used it before and nobody complained. 😬